ee/billing/billing_manager.py

```diff
@@ -57,6 +57,7 @@ def get_billing(self, organization: Optional[Organization], plan_keys: Optional[
         if organization and self.license and self.license.is_v2_license:
             billing_service_response = self._get_billing(organization)
 
+            raise Exception("something is wrong")
```

Comment:
Should this be removed?

Reply:
Yeah, sorry, left it in after my demo screen recording..Will remove.